### PR TITLE
Python 3: Use string type due to call of json_handler at stash-take.py

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

File: fetch/api/resources/stash-put.py [hunk 1]
```diff
@@ -9,6 +11,6 @@ def main(request, response):
     url_dir = u'/'.join(request.url_parts.path.split(u'/')[:-1]) + u'/'
     key = request.GET.first(b"key")
     value = request.GET.first(b"value")
-    request.server.stash.put(key, value, url_dir)
+    request.server.stash.put(key, isomorphic_decode(value), url_dir)
```
Comment (Hexcles) [inline fetch/api/resources/stash-put.py:14]: This coupling is a bit unfortunate and brittle, splitting the getter and setter of the same property across two different files.

Let's add a comment to point out `value` will be json.dump()'ed in stake-take.py so it must be a text string.

**Review by Hexcles — APPROVED** [review]
LGTM % requesting a comment

File: fetch/api/resources/stash-put.py (hunk 1 above)
Comment (ziransun) [reply fetch/api/resources/stash-put.py:15]: Comment added. Thank you!

File: fetch/api/resources/stash-put.py [hunk 2]
```diff
@@ -9,6 +11,7 @@ def main(request, response):
     url_dir = u'/'.join(request.url_parts.path.split(u'/')[:-1]) + u'/'
     key = request.GET.first(b"key")
     value = request.GET.first(b"value")
-    request.server.stash.put(key, value, url_dir)
+    # value here must be a text string. It will be json.dump()'ed in stake-take.py.
```
Comment (stephenmcgruer) [inline fetch/api/resources/stash-put.py:14]:
```suggestion
    # value here must be a text string. It will be json.dump()'ed in stash-take.py.
```

Comment (ziransun) [reply fetch/api/resources/stash-put.py]: Silly me! Updated. Thank you!

**Review by stephenmcgruer — APPROVED** [review]

**Pull request merged**